modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/publisher/APIM638ValidateTheRoleOfAnExistingUserThroughThePublisherRestAPITestCase.java

```diff
@@ -82,46 +81,32 @@ public void setEnvironment() throws Exception {
         //Create a new Role with Login and Create API Permission - API Publisher
         userManagementClientAdmin.addRole(creatorRole, new String[]{},
                 new String[]{permissionListApiLogin, permissionListApiCreate});
-
-        //Login to API Publisher with carbon super admin user
-        apiPublisher = new APIPublisherRestClient(publisherURLHttp);
-        apiPublisher.login(publisherContext.getContextTenant().getContextUser().getUserName(),
-                publisherContext.getContextTenant().getContextUser().getPassword());
-
     }
 
     @Test(groups = {"wso2.am"}, description = "Validate the Role of an existing user through" +
             " the publisher rest API ")
     public void testValidateTheRoleOfAnExistingUser() throws Exception {
-
         //Validate the admin role
-        JSONObject jsonObjectAdmin = new JSONObject
-                (apiPublisher.validateRoles(adminRole).getData());
-        assertFalse(jsonObjectAdmin.getBoolean("error"), "Invalid Role Name - " + adminRole);
-        assertTrue(jsonObjectAdmin.getBoolean("response"),
-                "Invalid response for the Role - " + adminRole);
+        ApiResponse<Void> apiResponse = restAPIPublisher.validateRoles(adminRole);
+        assertEquals(apiResponse.getStatusCode(), HttpStatus.SC_OK,
+                "Response code mismatch for: " + adminRole);
 
         //Validate the creator role
-        JSONObject jsonObjectCreator = new JSONObject
-                (apiPublisher.validateRoles(creatorRole).getData());
-        assertFalse(jsonObjectCreator.getBoolean("error"), "Invalid Role Name - " + creatorRole);
-        assertTrue(jsonObjectCreator.getBoolean("response"),
-                "Invalid response for the Role - " + creatorRole);
-
+        apiResponse = restAPIPublisher.validateRoles(creatorRole);
+        assertEquals(apiResponse.getStatusCode(), HttpStatus.SC_OK,
+                "Response code mismatch for: " + creatorRole);
     }
 
     @Test(groups = {"wso2.am"}, description = "Validate the Role of an non-existing user through" +
-            " the publisher rest API ")
+            " the publisher rest API ", dependsOnMethods = {"testValidateTheRoleOfAnExistingUser"})
```

Comment:
since the dependsOnMethods only has one method, you can remove the curly braces("{" and "}")

Reply:
Fixed with cf2570e